src/frontend/next/src/pages/myfeeds.tsx

```diff
@@ -0,0 +1,200 @@
+import { useState, useEffect, FormEvent } from 'react';
+import { useRouter } from 'next/router';
+import { Box, Card, Container, Grid, IconButton, Typography } from '@material-ui/core';
+import { AccountCircle, AddCircle, RssFeed } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+import { ValidatorForm, TextValidator } from 'react-material-ui-form-validator';
+import { isWebUri } from 'valid-url';
+import Head from 'next/head';
+
+import config from '../config';
+import ExistingFeedList from '../components/ExistingFeedList';
+import HelpPopoverButton from '../components/HelpPopoverButton';
+import CustomizedSnackBar from '../components/SnackBar';
+import { useUser } from '../components/UserProvider';
+import { User, Feed, FeedHash } from '../interfaces';
+
+const useStyles = makeStyles((theme) => ({
+  margin: {
+    margin: theme.spacing(2),
+  },
+  button: {
+    padding: '3px 0 3px 0',
+  },
+}));
+
+const MyFeeds = () => {
+  const router = useRouter();
+  const user: User = useUser();
```

Comment:
You shouldn't need to type this, it will get inferred (same as `useRouter()` and `useStyles()`)